src/main/java/by/issoft/dto/HttpDeleteWithBody.java

```diff
@@ -0,0 +1,18 @@
+package by.issoft.dto;
+
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+
+import java.net.URI;
+
+public class HttpDeleteWithBody extends HttpEntityEnclosingRequestBase {
```

Comment:
this is not DTO class. You can move it into Request class as a ` private static class` somewhere after send() method.

```
 // some code of Request class

public HttpResponse send() {
        try {
            return httpClient.execute((HttpUriRequest) request);
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }

    private static class HttpDeleteWithBody extends HttpEntityEnclosingRequestBase {

        @Override
        public String getMethod() {
            return "DELETE";
        }

        public HttpDeleteWithBody(String URL) {
            super();
            setURI(URI.create(URL));
        }
    }
}
```